### PR TITLE
new version of tracie with revised evaluator (now single file)

### PR DESCRIPTION
**Pull request by yakazimir**
rather than having two different sets to evaluate, we decided to go for a single evaluation (details can be found in the revised `README`s).

### CODE REVIEW

File: tracie/data/README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 This directory contains the training and test files for evaluating predictions,
 and a sample prediction file.
 
-The files `train_iid.jsonl` and `train_uniform.jsonl` are two versions of the
-training data that can be used. These two training sets refer to splits, based
-on a strict `iid` split and more balanced `uniform` split.
+The files `train_uniform.jsonl` is the main training data to use for
+leaderboard entries (please note that in our paper, we also experiment
+with training on an `iid`set (not included here) _**which is not
```
Comment (aimichal) [inline tracie/data/README.md]:
```suggestion
with training on an `iid` set (not included here) _**which is not
```

File: tracie/data/README.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 This directory contains the training and test files for evaluating predictions,
 and a sample prediction file.
 
-The files `train_iid.jsonl` and `train_uniform.jsonl` are two versions of the
-training data that can be used. These two training sets refer to splits, based
-on a strict `iid` split and more balanced `uniform` split.
+The files `train_uniform.jsonl` is the main training data to use for
```
Comment (aimichal) [inline tracie/data/README.md]:
```suggestion
The file `train_uniform.jsonl` is the main training data to use for
```

File: tracie/evaluator/evaluator.py [hunk 3]
```diff
@@ -97,20 +97,20 @@ def main():
...
-        "-train_type",
-        help="The particular training data used.",
-        required=True,
-    )
+    # parser.add_argument(
+    #     "--train_type",
+    #     "-train_type",
+    #     help="The particular training data used.",
+    #     required=True,
+    # )
```
Comment (aimichal) [inline tracie/evaluator/evaluator.py]: Delete unused lines.

```suggestion

```

File: tracie/evaluator/evaluator.py [hunk 4]
```diff
@@ -97,20 +97,20 @@ def main():
...
-    valid_train_sets = {"train_iid", "train_uniform"}
-    if args.train_type not in valid_train_sets:
-        raise ValueError(
-            "Training type must be from: %s" % ",".join(list(valid_train_sets))
-        )
+    # valid_train_sets = {"train_iid", "train_uniform"}
+    # if args.train_type not in valid_train_sets:
+    #     raise ValueError(
+    #         "Training type must be from: %s" % ",".join(list(valid_train_sets))
+    #     )
```
Comment (aimichal) [inline tracie/evaluator/evaluator.py]:
```suggestion
```

File: tracie/evaluator/evaluator.py [hunk 5]
```diff
@@ -120,7 +120,7 @@ def main():
         output.write(
             json.dumps(
                 {
-                    "train_type": args.train_type,
+                    #"train_type": args.train_type,
```
Comment (aimichal) [inline tracie/evaluator/evaluator.py]:
```suggestion
```

**Review by aimichal — APPROVED** [review]
I haven't run this evaluator, but I assume you did an it runs!

Minor suggestions inline.

**Comment by yakazimir** [comment]
Yes, it works fine (the outputs are as given in the READMEs), also inside of our internal Jetty app now (as we talked about).

**Pull request merged**